### DISCUSSION TITLE
Location of gauss points in global coordinate

### DISCUSSION THREAD

**Discussion by YuanxinXiao** [idaholab/moose#28564] [Q&A Modules: Solid mechanics]
Hi @GiudGiud ,

I am struggling to find out the location of gauss points in global coordinates as I want to assign material property to gauss points.

It would be great if you help me to do so

**Comment by GiudGiud** [comment]
hello

in which object?
quadrature points are found in `_q_point[_qp]` in many objects

**Comment by YuanxinXiao** [comment]
hi

I want to store the location of quadrature points in csv file

 Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by GiudGiud** [reply]
https://github.com/idaholab/moose/pull/28565 has a new Positions object that will output the quadrature points to JSON

**Reply by GiudGiud** [reply]
from JSON to csv should be easy with a python script
